app/controllers/feedbacks_controller.rb

```diff
@@ -32,4 +32,13 @@ def load_support_languages
     @support_languages = Language.where(support_available: true).order(:name)
   end
 
+  private
+
+  def feedback_params
+    params.require(:feedback).permit(
+      :comment, :email, :summary, :user_agent, :category, :approved,
```

Comment:
I'm not sure what approved is or was for, but category was from the old version of the Support form, and we've left the column in the database in case we need the old data.